lib/hair_trigger.rb

```diff
@@ -148,7 +148,7 @@ def generate_migration(silent = false)
 # While you can edit this file, any changes you make to the definitions here
 # will be undone by the next auto-generated trigger migration.
 
-class #{migration_name} < ActiveRecord::Migration
+class #{migration_name} < ActiveRecord::Migration[#{[ActiveRecord::VERSION::MAJOR, ActiveRecord::VERSION::MINOR].join(".")}]
```

Comment:
It looks like this is a pretty similar change to what was already merged into master:

https://github.com/jenseng/hair_trigger/blob/master/lib/hair_trigger.rb#L152

```
class #{migration_name} < ActiveRecord::Migration[#{ActiveRecord::Migration.current_version}]
```

@kwasimensah - we were having the same issue locally until the new version that was released today (2019-02-08 - [v0.2.22](https://github.com/jenseng/hair_trigger/releases/tag/v0.2.22)

Reply:
I forgot about this pull request :) Happy to drop if someone else already got this in.

Reply:
Yeah - I think it'd be fine to close this since it was changed here: https://github.com/jenseng/hair_trigger/commit/a9993362fc0238732effce543b86bf257801324b#diff-f290e98586ed7fa1a183ffb6da5c8a8dR152 

(Though definitely worth double-checking that it's working on your end too with the latest bump!)